challenges/objects-arrays.js

```diff
@@ -24,6 +45,9 @@ console.log(stegosaurus.length);
 console.log(tyrannosaurus.period);
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
+tyrannosaurus.roar = function () {
+  return 'RAWERSRARARWERSARARARRRR!';
+};
 console.log(tyrannosaurus.roar());
```

Comment:
good. I think I when I went through this assignment, I just went back to the object itself and created the roar method. Inline, for obvious reasons, looks better and works. Good job!